person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @name = name
```

Comment:
- [ ] Please add the instance variable `@id` as mentioned in the project requirements.

Reply:
@Lucas-Masaba alright, will do so